package-lock.json

```diff
@@ -31296,11 +31301,6 @@
       "resolved": "https://registry.npmjs.org/prepend-http/-/prepend-http-1.0.4.tgz",
       "integrity": "sha1-1PRWKwzjaW5BrFLQ4ALlemNdxtw="
     },
-    "prettier": {
```

Comment:
Please avoid committing changes to the package-lock when dependencies aren't changing as part of the issue.